### PR TITLE
PLANET-7311 Apply new visual identity

### PR DESCRIPTION
**Pull request by mleray**
### Description

See [PLANET-7311](https://jira.greenpeace.org/browse/PLANET-7311)

**Requirements**
- Images
  - Replace the old images with the new ones

- Typefaces
  - Apply the new GP Sans to all the headings on the website.
  - Make the Source Serif Pro the default body text.
  - Apply the Sources Sans 3 to the footer links.

- Colors
  - Apply the P4 Action Yellow to the TA buttons.
  - Apply the GP Green to the TA button in the navigation bar.
  - Apply the Dark green 800 & Dark green 900 to the footer.
  - Apply the Beige 100, GP Green 100, and Dark green 800 to the background of the blocks. (see mockups).

**Resources**

[Figma file](https://www.figma.com/file/Uwc7Ja0SIeZXDFhAewrFwn/P4-product-pages?type=design&node-id=1%3A4282&mode=design&t=xBtA7ghK2GDeNLGN-1)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
     "build-js": "uglifyjs --compress --output dist/app.js -- js/scripts.js js/scripts-features.js",
     "static-prod": "replace-in-file '/{{ base_url }}/g' 'https://storage.googleapis.com/planet4-homepage/' dist/index.html,dist/features/index.html --isRegex; replace-in-file '/{{ domain }}/g' 'https://planet4.greenpeace.org/' dist/index.html,dist/features/index.html --isRegex",
     "static-dev": "replace-in-file '/{{ base_url }}/g' '/' dist/index.html,dist/features/index.html --isRegex ; replace-in-file '/{{ domain }}/g' '/' dist/index.html,dist/features/index.html --isRegex",
-    "build": "npm run build-sass && npm run build-js && cp -r index.html img/ features/ dist/",
+    "build": "npm run build-sass && npm run build-js && cp index.html dist/ && cp -r img/. dist/img/ && cp -r features/. dist/features/",
```
Comment (mleray) [inline package.json:14]: Updated this command to work for Mac OS too

File: css/_features.scss [hunk 2]
```diff
@@ -30,28 +30,15 @@
...
-  }
-  @media screen and (min-width: $large-width) {
-    .section-title {
-      font-size: 2.5rem;
-    }
   }
 
   p {
-    font-family: $roboto, sans-serif;
+    font-family: $font-family-paragraph-primary;
```
Comment (GP-Dan-Tovbein) [inline css/_features.scss]: Can we use CSS variables instead?

File: css/_features.scss [hunk 3]
```diff
@@ -166,9 +153,9 @@
     position: absolute;
     top: 3px;
     padding: 1px 4px;
```
Comment (GP-Dan-Tovbein) [inline css/_features.scss]: Maybe replace `4px` with `$sp-x` ?

File: css/_footer.scss [hunk 4]
```diff
@@ -1,10 +1,14 @@
 /* Footer styles */
 #site-footer {
-  background-color: #042a40;
+  background-color: var(--p4-dark-green-900);
   color: $white;
-  font-family: $roboto;
+  font-family: $font-family-paragraph-primary;
```
Comment (GP-Dan-Tovbein) [inline css/_footer.scss]: Same here!

File: css/_header.scss [hunk 5]
```diff
@@ -157,10 +157,14 @@
     padding: 0.2rem;
```
Comment (GP-Dan-Tovbein) [inline css/_header.scss:157]: Not part of the scope but we can avoid using `rem` for spacing

File: css/_js-expando.scss [hunk 6]
```diff
@@ -24,7 +26,7 @@
 }
 
 .expando-link::after {
-  background-image: url("data:image/svg+xml,%3Csvg width='16' height='9' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='M15.225.76a.857.857 0 010 1.257L8.663 8.24a.974.974 0 01-1.326 0L.775 2.017a.857.857 0 010-1.257.974.974 0 011.325 0L8 6.354 13.9.76a.976.976 0 011.23-.08z' fill='%236F7376' fill-rule='evenodd'/%3E%3C/svg%3E");
+  background-image: url("data:image/svg+xml,%3Csvg width='16' height='9' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='M15.225.76a.857.857 0 010 1.257L8.663 8.24a.974.974 0 01-1.326 0L.775 2.017a.857.857 0 010-1.257.974.974 0 011.325 0L8 6.354 13.9.76a.976.976 0 011.23-.08z' fill='black' fill-rule='evenodd'/%3E%3C/svg%3E");
```
Comment (GP-Dan-Tovbein) [inline css/_js-expando.scss]: We can use the `grey-900` token value instead!

File: css/_sections.scss [hunk 7]
```diff
@@ -46,7 +46,11 @@
 
   .section-title {
     text-align: inherit;
-    font-size: $font-size-xxl;
+    font-size: $font-size-xl;
+
+    @include large-and-up {
+      font-size: $font-size-xxl;
```
Comment (GP-Dan-Tovbein) [inline css/_sections.scss]: We can use tokens instead

**Review by GP-Dan-Tovbein — COMMENTED** [review]
Code looks good to me!

However, I've left some comments about encourage to replace variables with tokens and SASS variables with CSS variables.

**Pull request merged**